### PR TITLE
Add the schema OpenSiteDraft.ecschema.xml

### PR DESCRIPTION
**Pull request by FelixGirard**
Add the "draft" schema for OpenSite that represent basic shapes for drafting purposes.

### CODE REVIEW

File: Domains/4-Application/OpenSite/OpenSiteDraft.ecschema.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="OpenSiteDrafts" alias="OpenSiteDrafts" version="01.00.00" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2" displayLabel="OpenSite Drafts" description="Drafts schema for drawing precise shapes in OpenSite+.">
+    <ECSchemaReference name="CoreCustomAttributes" version="01.00.04" alias="CoreCA" />
+    <ECSchemaReference name="BisCore" version="01.00.10" alias="bis" />
+    <ECSchemaReference name="RoadRailUnits" version="01.00.01" alias="rru" />
+
+    <ECEntityClass typeName="Draft" displayLabel="Draft">
+        <BaseClass>bis:GeometricElement3d</BaseClass>
```
Comment (diegoalexdiaz) [inline Domains/4-Application/OpenSite/OpenSiteDraft.ecschema.xml:12]: Please use bis:SpatialLocationElement as base-class instead.

File: Domains/4-Application/OpenSite/OpenSiteDraft.ecschema.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="OpenSiteDrafts" alias="OpenSiteDrafts" version="01.00.00" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2" displayLabel="OpenSite Drafts" description="Drafts schema for drawing precise shapes in OpenSite+.">
+    <ECSchemaReference name="CoreCustomAttributes" version="01.00.04" alias="CoreCA" />
+    <ECSchemaReference name="BisCore" version="01.00.10" alias="bis" />
+    <ECSchemaReference name="RoadRailUnits" version="01.00.01" alias="rru" />
+
```
Comment (diegoalexdiaz) [inline Domains/4-Application/OpenSite/OpenSiteDraft.ecschema.xml:10]: Please add the following Custom-Attribute to advertise the status and layer of this schema:

```
   <ECCustomAttributes>
        <ProductionStatus xmlns="CoreCustomAttributes.01.00.03">
            <SupportedUse>NotForProduction</SupportedUse>
        </ProductionStatus>
        <SchemaLayerInfo xmlns="BisCustomAttributes.01.00.00">
            <Value>Application</Value>
        </SchemaLayerInfo>
    </ECCustomAttributes>
```
Note that this addition will force you to add BisCustomAttributes as a reference to this schema.

**Comment by diegoalexdiaz** [comment]
Please update the SchemaInventory by following the steps at: https://github.com/iTwin/bis-schemas/blob/master/README.md#update-schema-inventory.

File: Domains/4-Application/OpenSite/OpenSiteDraft.ecschema.xml (hunk 1 above)
Comment (FelixGirard) [reply Domains/4-Application/OpenSite/OpenSiteDraft.ecschema.xml:12]:
```suggestion
        <BaseClass>bis:GraphicalElement3d</BaseClass>
```

File: Domains/4-Application/OpenSite/OpenSiteDraft.ecschema.xml (hunk 2 above)
Comment (diegoalexdiaz) [reply Domains/4-Application/OpenSite/OpenSiteDraft.ecschema.xml:11]: Please look at the OpenSite schema as an example of how this Custom-Attribute is applied: https://github.com/iTwin/bis-schemas/blob/master/Domains/4-Application/OpenSite/OpenSite.ecschema.xml

**Review by diegoalexdiaz — APPROVED** [review]

**Pull request closed without merging**